test/JDBC/upgrade/master/schedule

```diff
@@ -259,7 +259,6 @@ sys-schema-name
 sys-schemas
 sys-schemas-dep
 sys-selective_xml_index_paths
-sys-server_principals
```

Comment:
Why this change?

Reply:
I have created logins in prepare testfile and logins are not dumped in case of database-level dump/restore so that's why removed this test from master/schedule file